corehq/apps/data_dictionary/management/commands/populate_case_prop_groups.py

```diff
@@ -25,6 +25,9 @@ def populate_case_prop_groups(domain):
     case_props = CaseProperty.objects.exclude(group__exact="").filter(**filter_kwargs)
 
     for case_prop in case_props:
-        group = CasePropertyGroup.get_or_create(case_prop.group, domain, case_prop.case_type)
+        group, created = CasePropertyGroup.objects.get_or_create(
+            name=case_prop.group,
+            case_type=case_prop.case_type
+        )
```

Comment:
is this the intended change for this commit?
" Refactor to use CasePropertyGroup.objects.get_or_create "

Reply:
Yeah, i have used the django inbuilt get_or_create method, i will remove the `get_or_create` method that i created as it is not needed.

Reply:
Ah, right. I missed that "objects" was now added.